### PR TITLE
LB-261: Show total listen count on index page ala AB

### PR DESCRIPTION
**Pull request by paramsingh**
# Summary

* This is a…
    * [ ] Bug fix
    * [x] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other

# Problem

* JIRA ticket (_optional_): [LB-261](https://tickets.metabrainz.org/browse/LB-261)

# Solution

Add a table to which we can pass stats.

This table should also open the doors to showing
other stats such as artist count etc on the home
page.

### CODE REVIEW

**Comment by paramsingh** [comment]
screenshot_2017-12-21_00-06-21

File: listenbrainz/webserver/views/index.py [hunk 1]
```diff
@@ -23,7 +23,19 @@
 
 @index_bp.route("/")
 def index():
-    return render_template("index/index.html")
+
+    # get total listen count
+    try:
```
Comment (mayhem) [inline listenbrainz/webserver/views/index.py]: I think we ought to add some caching to this -- maybe for 2-3 minutes? I'd hate to have to make this (albeit small) query for loading the home page each time.

Comment (paramsingh) [reply listenbrainz/webserver/views/index.py]: `get_total_listen_count` already does caching. 

https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/listenstore/influx_listenstore.py#L120

Comment (mayhem) [reply listenbrainz/webserver/views/index.py]: Oh right. Grand. :)

**Pull request merged**